bin/cfg/Tomb1Main.json5

```diff
@@ -185,6 +185,9 @@
     // Disables music in the main menu.
     "disable_music_in_menu": false,
 
+    // Pauses sound in the inventory screen.
+    "disable_inventory_sound": false,
```

Comment:
Maybe let's name it `disable_game_sound_in_inventory` to make sure nobody reads it as "disable all sounds including inventory ring sounds"...